.github/ISSUE_TEMPLATE.md

```diff
@@ -1,28 +1,42 @@
+**PLEASE READ:** Before reporting here, *please* carefully read our **Troubleshooting Guide** at <https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ> and search the issues page here for your error/problem, as most postd bugs are duplicates or easy fixes.
+
+If you do post an issue here, please describe step by step in detail what you were doing when the error occurred,. Otherwise, your issue will likely be closed after 7 days. Thanks.
+
+**Note:** Delete this message before submitting.
+
 ## Description of your problem
 
-**What steps will reproduce the problem?**
+
+
+## What steps will reproduce the problem?
 
 1. 
 2. 
 3. 
 
-**What is the expected output? What do you see instead?**
+## What is the expected output? What do you see instead?
+
+
+
+### Paste Traceback/Error Below (if applicable)
```

Comment:
I don't think this section should be part of the issue template because that's why we have our new error dialog.

Reply:
This is just for the template, for users who report manually via Github, not our automatic system (e.g. if the error occurs on Spyder startup, they can't get it to launch at all, or they temporarily/permanently disabled the error dialog). Many of those cases would likely have a traceback, or at least an error message, and this would make it clear we need it if so. Let me know if you still want me to remove it.

Reply:
Ok, let's leave this section for now.